src/detect-byte-extract.c

```diff
@@ -305,7 +313,13 @@ static inline DetectByteExtractData *DetectByteExtractParse(const char *arg)
                            "for arg %d for byte_extract", i);
                 goto error;
             }
-            int multiplier = atoi(multiplier_str);
+            int multiplier;
```

Comment:
Should `multiplier` be declared as `int32` instead of `int` ?

Reply:
Yes, sure.